timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -448,13 +453,43 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         this.isNavigating = flag;
     }
 
+    setEventClickedFlag(flag: boolean) {
+        this.isEventClicked = flag;
+    }
+
+    private selectStateOnEventClicked() {
+        const selectedRow = this.rows.find(row => {
+            if (row && this.unitController.selectionRange) {
+                const cursorPosition = this.unitController.selectionRange.end;
+                const rowElement = row.states.find((rowElementModel: TimelineChart.TimeGraphState) => rowElementModel.range.start === cursorPosition
+                    || rowElementModel.range.end === cursorPosition);
+                if (rowElement) {
+                    return true;
+                }
+            }
+        });
+
+        if (selectedRow) {
+            const actualRow = this.rowComponents.get(selectedRow);
+            const chartY = 0 - this.layer.position.y;
+            const rowVerticalOffset = actualRow ? actualRow.position.y : 0;
+            const rowHeight = actualRow ? actualRow.height : 0;
+
+            if (rowVerticalOffset < chartY) {
+                this.rowController.verticalOffset = rowVerticalOffset;
+            } else if (rowVerticalOffset > chartY + this.layer.height) {
+                this.rowController.verticalOffset += rowVerticalOffset - (chartY + this.layer.height) + rowHeight;
+            }
+        }
+    }
+
     protected selectStateInNavigation() {
         const row = this.rowController.selectedRow;
         if (row && this.unitController.selectionRange) {
             const cursorPosition = this.unitController.selectionRange.end;
-            const rowElement = row.states.find((rowElementModel: TimelineChart.TimeGraphState) => rowElementModel.range.start === cursorPosition || rowElementModel.range.end === cursorPosition);
+            const rowElement = row.states.find((rowElementModel: TimelineChart.TimeGraphState) => rowElementModel.range.start === cursorPosition
+                || rowElementModel.range.end === cursorPosition);
             this.selectRowElement(rowElement);
         }
-        this.setNavigationFlag(false);
```

Comment:
This line was accidentally removed

Reply:
It's no longer required after #67. But I decided not to modify this file as part of this PR.